chat_shell/tests/test_skill_mcp_loader.py

```
@@ -434,3 +434,39 @@ async def test_return_type_is_tuple(self):
+        mock_registry.create_tools_for_skill.side_effect = _capture_context
+
+        with patch(
+            "chat_shell.skills.SkillToolRegistry.get_instance",
+            return_value=mock_registry,
+        ):
+            await prepare_skill_tools(
+                task_id=1,
+                subtask_id=2,
+                user_id=3,
+                skill_configs=skill_configs,
+                skill_identity_token="skill-jwt",
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to argument: "skill_identity_token"

(S106)